etc/config/ispc.amazon.properties

```
@@ -1,10 +1,13 @@
 compilers=&ispc
-defaultCompiler=ispc1281
+defaultCompiler=ispc1282
 objdumper=/opt/compiler-explorer/gcc-14.2.0/bin/objdump
 
-group.ispc.compilers=ispc1281:ispc1280:ispc1270:ispc1260:ispc1253:ispc1250:ispc1240:ispc1230:ispc1220:ispc1210:ispc1200:ispc1190:ispc1180:ispc1170:ispc1161:ispc1160:ispc1150:ispc1141:ispc1140:ispc1130:ispc1120:ispc1110:ispc1100:ispc192:ispc191:ispc-trunk
+group.ispc.compilers=ispc1282:ispc1281:ispc1280:ispc1270:ispc1260:ispc1253:ispc1250:ispc1240:ispc1230:ispc1220:ispc1210:ispc1200:ispc1190:ispc1180:ispc1170:ispc1161:ispc1160:ispc1150:ispc1141:ispc1140:ispc1130:ispc1120:ispc1110:ispc1100:ispc192:ispc191:ispc-trunk
 group.ispc.isSemVer=true
 group.ispc.baseName=ispc
+compiler.ispc1281.exe=/opt/compiler-explorer/ispc-1.28.2/bin/ispc
+compiler.ispc1281.semver=1.28.2
+compiler.ispc1281.executableLinker=/opt/compiler-explorer/gcc-14.2.0/bin/gcc
```

These refer to iscp1281 still (the CI tests caught this)


```suggestion
compiler.ispc1282.exe=/opt/compiler-explorer/ispc-1.28.2/bin/ispc
compiler.ispc1282.semver=1.28.2
compiler.ispc1282.executableLinker=/opt/compiler-explorer/gcc-14.2.0/bin/gcc
```